backend/WebApi/WebApiKernel.cs

```diff
@@ -11,15 +10,15 @@ public static void Setup(IServiceCollection services)
         services.AddHttpClient();
         services.AddSingleton<ProjectService>();
         services.AddSingleton<UserService>();
-        //todo should be built from JWT
-        services.AddSingleton(new WebUserContext
+        services.AddHttpContextAccessor();
+        services.AddScoped((serviceProvider) =>
```

Comment:
instead of using a factory function here, maybe we should just inject `IHttpContextAccessor` into `LfWebContext`, just so we don't need to update code here if we need to add something to `LfWebContext`

Reply:
I like that idea except for the fact that LfWebContext would then be referencing the JWTService. I'd prefer it to just be given data. But...maybe I'll try it.

Reply:
Yeah, I think it looks nice and tidy 🤓, you're right.

Reply:
I like it! Now if we really wanted we could do something like pass the active project via a request header and set the active project code in LfWebContext.